ipatests/test_integration/test_caless.py

```diff
@@ -1492,6 +1492,25 @@ def test_ds_old_options(self):
                                   args=args, stdin_text=stdin_text)
         assert_error(result, "no such option: --dirsrv-pin")
 
+    def test_anon_pkinit_with_external_CA(self):
+
+        ca2_fname = os.path.join(self.cert_dir, 'ca2.pem')
+        self.prepare_cacert('ca2', ca2_fname)
+        self.copy_cert(self.master, 'ca2.pem')
```

Comment:
Shouldn't this be ca2_fname instead of 'ca2.pem'? For each reference to 'ca2.pem'?

Reply:
Aah copy_cert is already taking care of paths. So now it should be more clear.